WooCommerce/Classes/Authentication/Store Creation/StoreCreationCoordinator.swift

```diff
@@ -680,6 +684,14 @@ private extension StoreCreationCoordinator {
                     DDLogInfo("🔵 Retrying: Site available but is not a jetpack site yet...")
                     return continuation.resume(throwing: StoreCreationError.newSiteIsNotJetpackSite)
                 }
+
+                // Sometimes, as soon as the jetpack installation is done some properties like `name` and `isWordPressComStore` are outdated.
+                // In this case, let's keep retrying sites syncing. https://github.com/woocommerce/woocommerce-ios/pull/9317#issuecomment-1488035433
+                guard site.isWordPressComStore && site.isWooCommerceActive && site.name == expectedStoreName else {
+                    DDLogInfo("🔵 Retrying: Site available but properties are not yet in sync...")
+                    return continuation.resume(throwing: StoreCreationError.newSiteIsNotJetpackSite)
+                }
```

Comment:
@jaclync not expecting a full review from you but just tagging you here in case you have some opinions on this change!

Reply:
This looks good to me 👍 ideally, this can be addressed on the server side. let's hope it doesn't take more than a minute or so for the majority of the merchants 😅 

Maybe we can also track the duration in the `siteCreated` event as an event property, if it becomes an issue later on.